covfee/client/models/Node.ts

```diff
@@ -30,6 +30,15 @@ export function useNodeFns(node: NodeType) {
     [node.url]
   )
 
+  const submitProgress = useCallback(
+    (progress: number) => {
+      const url = node.url + "/progress/" + progress
+
+      return fetcher(url).then(throwBadResponse)
```

Comment:
I'm a bit confused, doesn't this do the `throwBadResponse` when the url is fetched?